3001-4000/LIT3592Miracle.xml

```diff
@@ -92,81 +100,12 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?><TEI
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML4205">
-                            <desc>see 
-                               <ref type="mss" corresp="EMML4205">EMML 4205, ff. 25v, 51r, 26r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2378">
-                            <desc>
-                               <ref type="mss" corresp="EMML2378">EMML 2378, f. 31r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML6640">
-                            <desc>
-                               <ref type="mss" corresp="EMML6640">EMML 6640, f. 19r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2059">
-                            <desc>
-                               <ref type="mss" corresp="EMML2059">EMML 2059, f. 135r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2060">
-                            <desc>
-                               <ref type="mss" corresp="EMML2060">EMML 2060, f. 183r</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2066">
-                            <desc>
-                               <ref type="mss" corresp="EMML2066">EMML 2066, f.  60r, 115v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML2802">
-                            <desc>
-                               <ref type="mss" corresp="EMML2802">EMML 2803, f. 66v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML6196">
-                            <desc>
-                               <ref type="mss" corresp="EMML6196">EMML 6196, f. 19v</ref>
-                            </desc>
-                         </relation>
-                         <relation name="lawd:hasAttestation" active="LIT3592Miracle" passive="EMML7543">
-                            <desc>
-                               <ref type="mss" corresp="EMML7543">EMML 7543, f. 27r</ref>
-                            </desc>
-                         </relation>
+               <relation name= "saws:isVersionOf" active="LIT3592Miracle" passive="LIT7150Miracle"/>
```

Comment:
the relation goes only the other way around

7150/7151 is a version of 3592

since the versions have the correct syntax the relations must be removed here